pkg/cmd/codespace/ssh.go

```diff
@@ -44,6 +46,11 @@ func newSSHCmd(app *App) *cobra.Command {
 		Long: heredoc.Doc(`
 			The 'ssh' command is used to SSH into a codespace. In its simplest form, you can
 			run 'gh cs ssh', select a codespace interactively, and connect.
+			
+			By default, the 'ssh' command will create a public/private ssh key pair to  
+			authenticate with the codespace (if they haven't already been created) inside the 
```

Comment:
We could take out `if they haven't already been created` to simplify this section of the docs. The only thing that the user needs to know is that a keypair will automatically be generated as needed—the details of how this is done doesn't have to be known to the user. We could also omit the destination directory where the keypair will be stored.

Reply:
Updated